docs/operate/reference/module-configuration.md

```diff
@@ -234,15 +234,22 @@ If, for example, the module provides a motor component, and the motor is running
 
 #### Environment variables
 
-Each module has access to the following default environment variables:
+Each module has access to the following default environment variables.
+Not all of these variables are available on [local modules](/operate/get-started/other-hardware/#test-your-module-locally); you can manually set variables your module requires if necessary.
 
 <!-- prettier-ignore -->
-| Name | Description |
-| ---- | ----------- |
-| `VIAM_HOME` | The root of the `viam-server` configuration.<br>Default: `$HOME/.viam` |
-| `VIAM_MODULE_ROOT` | The root of the module install directory. The module process uses this directory as its current working directory (`cwd`). This variable is useful for file navigation that is relative to the root of the module. If you are using a [local module](/operate/get-started/other-hardware/#test-your-module-locally), you must set this value manually if your module requires it.<br>Example: `/opt/my-module/verxxxx-my-module/` |
-| `VIAM_MODULE_DATA` | A persistent folder location a module can use to store data across reboots and versions. This location is a good place to store [python virtual environments](/dev/reference/sdks/python/python-venv/).<br>Example: `$VIAM_HOME/module-data/cloud-machine-id/my-module-name/` |
-| `VIAM_MODULE_ID` | The module ID of the module. <br>Example: `viam:realsense` |
+| Name | Description | Automatically set on local modules? |
+| ---- | ----------- | ----------------------------------- |
+| `VIAM_HOME` | The root of the `viam-server` configuration.<br>Default: `$HOME/.viam` | <p class="center-text"><i class="fas fa-check" title="yes"></i></p> |
+| `VIAM_MODULE_ROOT` | The root of the module install directory. The module process uses this directory as its current working directory (`cwd`). This variable is useful for file navigation that is relative to the root of the module.<br>Example: `/opt/my-module/verxxxx-my-module/` | <p class="center-text"><i class="fas fa-check" title="yes"></i></p> |
+| `VIAM_MODULE_DATA` | A persistent folder location a module can use to store data across reboots and versions. This location is a good place to store [python virtual environments](/dev/reference/sdks/python/python-venv/).<br>Example: `$VIAM_HOME/module-data/cloud-machine-id/my-module-name/` | |
+| `VIAM_MODULE_ID` | The module ID of the module.<br>Example: `viam:realsense` | |
```

Comment:
```suggestion
| `VIAM_MODULE_ID` | The module ID of the module. Not available on local modules.<br>Example: `viam:realsense` | |
```

Reply:
The line doesn't have a check mark in the "automatically set" column--I'd think that's enough?

Reply:
Ah good, thanks